lib/science_wire_client.rb

```diff
@@ -287,6 +287,7 @@ def validate_query_id_response_xml(xml_doc)
         !xml_doc.xpath('//queryResultRows').empty?
     end
 
+    # @return [Array<Hash>]
     def send_query_and_return_pub_hashes(xml_query)
       xml_results = query_sciencewire(xml_query)
       xml_results.xpath('//PublicationItem').map do |sw_xml_doc|
```

Comment:
:ok: - c9ab3da8153d1fca239f210f04ae4e1d0414557a approved (without checking that that's what it actually returns, assuming that it's correct)